test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
if that application in the future got a front-end there are will be a validation for that user id will match to integer, but i dont think that will happens